components/network/tor/Makefile

```diff
@@ -15,18 +15,20 @@
 
 BUILD_BITS=		64
 
+USE_OPENSSL11=		yes
+
```

Comment:
Out build server hasn't the needed openssl mediator settings yet.
So if we merge now we'll get binaries dependant on openssl-1.0.2.
You either have to change it (and fix the REQUIRED_PACKAGES) and I can add the openssl-1.1 changes to my big PR or we'll need to wait until we changed the mediator settings on the build server.

Reply:
Hello,
FYI
show PR #6587